examples/composition-api/basic/pages/index.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="container">
+    <button @click="increment">
+      Increment
+    </button>
+    <p>Counter : {{ counter }}</p>
+    <button @click="decrement">
+      Decrement
+    </button>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { defineComponent, ref } from '@vue/composition-api'
+
+export default defineComponent({
+  name: 'CompositionAPICounter',
```

Comment:
Remove cause we're defining `index.vue` which is page component.
Also Nuxt automatically name component based on file name so it isn't required here
```suggestion
```